recipes/openssl/1.x.x/conanfile.py

```diff
@@ -436,9 +436,14 @@ def _get_env_build(self):
             self._env_build = AutoToolsBuildEnvironment(self)
         return self._env_build
 
+    def _get_default_openssl_dir(self):
+        if self.settings.os == "Linux" and self._full_version >= "1.1.0":
```

Comment:
what about MacOSX, FreeBSD, do they need the same?

Reply:
Very good question. Since I have a mac I just installed Conan there and tried a small test project with OpenSSL 1.1.1i and I have seen no issue. So I don't think we need to update the default "openssldir" variable there.

For FreeBSD and Android, I don't know.

Reply:
This certainly fixes Linux, no other platform is affect so I think we can continue with merging